server/odc-service/src/main/java/com/oceanbase/odc/service/connection/table/TableService.java

```diff
@@ -103,6 +113,12 @@ public class TableService {
     @Autowired
     private DBResourcePermissionHelper dbResourcePermissionHelper;
 
+    @PostConstruct
+    public void init() {
+        OB_MYSQL_SUPPORT_EXTERNAL_TABLE_VERSION = initDBSupportExternalTableVersion(DialectType.OB_MYSQL);
```

Comment:
check it every time when we calling `list` method, don't init it in postconstruct

Reply:
ok